x-pack/plugins/maps/public/components/map/mb/utils.js

```diff
@@ -28,6 +28,8 @@ export async function createMbMapInstance(node, initialView) {
     const mbMap = new mapboxgl.Map(options);
     mbMap.dragRotate.disable();
     mbMap.touchZoomRotate.disableRotation();
+    mbMap.addControl(
+      new mapboxgl.NavigationControl({ showCompass: false }), 'top-left');
```

Comment:
formatting is odd. just put on one line or make the brackets go all pretty and lined up.

Reply:
I'll opt for brackets lined up. I'm one of those "ideally <= 80 char width" weirdos